webapp/channels/src/components/common/hooks/useGetMultiplesExceededCloudLimit.test.ts

```diff
@@ -8,6 +8,7 @@ import useGetMultiplesExceededCloudLimit from './useGetMultiplesExceededCloudLim
 import type {LimitSummary} from './useGetMultiplesExceededCloudLimit';
 
 jest.mock('react', () => ({
+    ...jest.requireActual('react'),
```

Comment:
It needs it because the hook is called outside of a React context. To properly test this, we need [Testing Library's `renderHook` method](https://testing-library.com/docs/react-testing-library/api/#renderhook), but unfortunately that's on a newer version of the library which might require React 18